lib/shelley/exe/cardano-wallet.hs

```diff
@@ -293,14 +296,16 @@ withTracers
     -> (Trace IO MainLog -> Tracers IO -> IO a)
     -> IO a
 withTracers logOpt action =
-    withLogging [LogToStdout (loggingMinSeverity logOpt)] $ \(_, tr) -> do
+    withLogging [LogToStdout (loggingMinSeverity logOpt)] $ \(sb, (cfg, tr)) -> do
+        EKG.plugin cfg tr sb >>= loadPlugin sb
```

Comment:
What does this do?

This does not affect the integration tests. And you don't have a similar call in the integration tests. Do both work as intended?

Reply:
Yes, the integration tests don't need EKG/prometheus logging. That was the point of this change.